Packs/Cisco-umbrella-cloud-security/Integrations/Cisco-umbrella-cloud-security/Cisco-umbrella-cloud-security.py

```diff
@@ -29,9 +33,18 @@ def get_destination_lists(client: Client, **args) -> CommandResults:
 
     r = client._http_request('GET', uri)
 
-    data = {x['name']: x['id'] for x in r['data']}
+    data = []
+    for destination_list in r['data']:
```

Comment:
even though this change is probably the better way to go, this is a breaking change so need to mention it in the release notes

Reply:
RN: "get_destination_lists: context output like {'name':<name>,'id':<id>} instead of {<name>:<id>}"